.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: python
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27
     - python: 3.6
       env: TOXENV=py36
-    - python: 2.7  # TODO: bump to 3.6
```

Comment:
What happened here? I found it in commit 2fc9aba575af4911c11d3948b6627f9bcea6ca8a and got it :smirk:

Reply:
Obsolete now